pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -928,6 +930,13 @@ async def run(self):
 async def ocp4(runtime: Runtime, version: str, assembly: str, data_path: str, data_gitref: str, pin_builds: bool,
                build_rpms: str, rpm_list: str, build_images: str, image_list: str, skip_plashets: bool,
                mail_list_failure: str, ignore_locks: bool):
+
+    try:
+        job_id = os.environ['JOB_ID']
```

Comment:
instead of using env var JOB_ID, I think it is better to use something like `Runtime.get_job_run_name` so our code would be more independent from Jenkins environment.

Reply:
As pointed out [here](https://github.com/openshift-eng/aos-cd-jobs/pull/3935#issuecomment-1711429318), the lock identifier must be fed from Jenkins, because that's where the clean up procedure must be initiated. We can use a better env var name if we think `JOB_ID` is not appropriate, but it still needs to be set by Jenkins.

Reply:
resolving this, as this is outdated now